game/entities/sdCrystal.js

```diff
@@ -461,7 +461,7 @@ class sdCrystal extends sdEntity
 						reward_amount = Math.max( reward_amount, sdEntity.SCORE_REWARD_BROKEN_BIG_CRAB_CRYSTAL );
 					}
 					else
-					if ( this.is_anticrystal )
+					if ( this.is_anticrystal || this.matter_max >= 180000 || this.matter_regen >= 450 ) // Too high matter and regeneration will crash the server ( Preset Editor )
```

Comment:
It does sound more like a score shard spam, it probably would be best to address it where it is done. Probably with something like multi-score score shards that don't exist really.